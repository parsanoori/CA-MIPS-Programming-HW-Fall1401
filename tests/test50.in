57
174909673
299897966
572300398
116321894
467341668
507090367
246402560
213470515
880155973
745736509
195683242
1006075626
766796903
251976171
908064054
67441593
170737641
399109364
238735313
572666117
747367990
774687193
136683432
960795716
97695244
620452721
355408272
345385152
944674979
175456024
177248696
498600016
1009603070
768778947
538572355
21697291
126105095
1011115090
510553838
136657244
110346703
360215911
216057444
788678263
612070734
235145214
836251419
1030383123
876981672
437792571
833030128
829615225
708755444
860509894
729894617
963518344
672344776