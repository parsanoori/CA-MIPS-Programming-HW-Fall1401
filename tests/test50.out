21697291
67441593
97695244
110346703
116321894
126105095
136657244
136683432
170737641
174909673
175456024
177248696
195683242
213470515
216057444
235145214
238735313
246402560
251976171
299897966
345385152
355408272
360215911
399109364
437792571
467341668
498600016
507090367
510553838
538572355
572300398
572666117
612070734
620452721
672344776
708755444
729894617
745736509
747367990
766796903
768778947
774687193
788678263
829615225
833030128
836251419
860509894
876981672
880155973
908064054
944674979
960795716
963518344
1006075626
1009603070
1011115090
1030383123